pyeudiw/tests/tools/specifications.yml

```diff
@@ -0,0 +1,16 @@
+user_claims:
+  # Make only first two elements SD - this array will have three elements in the resulting SD-JWT, first two hidden
+  nationalities: 
+    - !sd "US"
```

Comment:
this doesn't make any sense, please use the attributes defined in the italian specs, pid-data-model.rst